Algorithms/Other/GeoLocation.cs

```diff
@@ -0,0 +1,50 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Algorithms.Other
+{
+    public class GeoLocation
+    {
+        private static readonly double EarthRadius = 6371.01d;
```

Comment:
Codacy found an issue: [Replace this 'static readonly' declaration with 'const'.](https://app.codacy.com/manual/siriak/C-Sharp/pullRequest?prid=6396283)